include/oneapi/dpl/pstl/hetero/utils_hetero.h

```diff
@@ -116,13 +116,11 @@ struct __identity_reduce_fn
     {
         using ::std::get;
         auto __chosen_for_min = __a;
-        auto __chosen_for_max = __b;
```

Comment:
Why was this logic changed? I think the original version was easier to read and would omit the if not check.

Reply:
I tried it both ways round, and found that this was slightly faster. I'd be interested to know if you find the same?

Reply:
I don't see runtime changes on A100 or PVC. How much difference did you observe? I would suggest keeping the original logic if it is not significantly faster. Also, the runtime might depend on the structure of the input data and which paths are taken in this comparison.